winui/AvatarView/content-types.md

```diff
@@ -1,10 +1,267 @@
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView ContentType="Initials"
+                         AvatarSize="ExtraLarge"
+                         AvatarName="Alex"
+                         InitialsType="DoubleCharacter"
+                         Background="Bisque">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+          
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.AvatarName = "Alex";
+avatarView.ContentType = AvatarContentType.Initials;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.InitialsType = AvatarInitialsType.DoubleCharacter;
+avatarView.Background = new SolidColorBrush(Colors.Bisque);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with DoubleCharacter](avatarview_images/winui_doubleinitials_avatarview.png)
+
+## Custom Image
+
+You can add a custom image by setting the `ImageSource` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView ContentType="CustomImage"
+                         AvatarSize="ExtraLarge"
+                         ImageSource="Images\person.png">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %} 
+{% highlight C# %}
+
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.ContentType = AvatarContentType.CustomImage;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.ImageSource = new BitmapImage(new Uri("ms-appx:///Images\\person.png"));
+           
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom image.](avatarview_images/winui_imagesource_avatarview.png)
+
+## Avatar Character
+
+You can set the pre-defined images in avatar view by setting the `AvatarCharacter` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView ContentType="AvatarCharacter"
+                         AvatarSize="ExtraLarge"
+                         AvatarCharacter="Avatar15">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+            
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.ContentType = AvatarContentType.AvatarCharacter;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.AvatarCharacter = AvatarCharacter.Avatar15;
+            
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with avatar character](avatarview_images/winui_avatarcharacter_avatarview.png)
+
+## Group View 
+
+You can add upto three images or initials in the same view using a GroupView type.
+
+`InitialsMemberPath`: Displays the initials in the group view.
+`ImageSourceMemberPath`: Displays the images in the group view.
+`BackgroundColorMemberPath`: Sets the background color of the group view.
+`InitialsColorMemberPath`: Sets the color for the initials in the group view.
+
+{% tabs %}
+{% highlight c# %}
+
+public class Employee
+{
+   private string name;
+
+   public string Name
```

Comment:
@MadhumithanJayapathi use single to declare the property like below.

 public Color Background { get; set; }

Reply:
@Manivannan-E Used the property as mentioned.